File: app/src/main/java/com/v1rex/smartincubator/ViewHolder/MentorViewHolder.java
package com.v1rex.smartincubator.ViewHolder;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.TextView;

import com.v1rex.smartincubator.R;

public class MentorViewHolder extends RecyclerView.ViewHolder {
    private TextView mNameTextView, mSpecialityTextView,mCityTextView;
    View mView ;
    public MentorViewHolder(View itemView) {
        super(itemView);
        mView = itemView;
        mNameTextView = (TextView) mView.findViewById(R.id.mentor_name_text_view);
        mCityTextView = (TextView) mView.findViewById(R.id.mentor_city_text_view);
        mSpecialityTextView = (TextView) mView.findViewById(R.id.mentor_speciality_text_view);

    }

    public void setmNameTextView(String name){
        mNameTextView.setText(name);
    }

    public void setmCityTextView(String city){
        mCityTextView.setText("City: " +  city);
    }

    public void setmSpecialityTextView(String speciality){
        mSpecialityTextView.setText("Speciality: " + speciality);
    }
}

File: app/src/main/java/com/v1rex/smartincubator/ViewHolder/MentorViewHolder.kt
package com.v1rex.smartincubator.ViewHolder

import android.content.Context
import android.support.v7.widget.RecyclerView
import android.view.View
import android.widget.TextView
import com.bumptech.glide.Glide
import com.bumptech.glide.load.engine.DiskCacheStrategy
import com.bumptech.glide.request.RequestOptions
import com.google.firebase.storage.StorageReference

import com.v1rex.smartincubator.R
import de.hdodenhof.circleimageview.CircleImageView

class MentorViewHolder(internal var mView: View) : RecyclerView.ViewHolder(mView) {
    private val mNameTextView: TextView
    private val mSpecialityTextView: TextView
    private val mCityTextView: TextView
    private val mImageProfile : CircleImageView

    init {
        mNameTextView = mView.findViewById<View>(R.id.mentor_name_text_view) as TextView
        mCityTextView = mView.findViewById<View>(R.id.mentor_city_text_view) as TextView
        mSpecialityTextView = mView.findViewById<View>(R.id.mentor_speciality_text_view) as TextView
        mImageProfile = mView.findViewById<View>(R.id.mentor_profile_image_view) as CircleImageView

    }

    fun setmNameTextView(name: String) {
        mNameTextView.text = name
    }

    fun setmCityTextView(city: String) {
        mCityTextView.text = "City: $city"
    }

    fun setmSpecialityTextView(speciality: String) {
        mSpecialityTextView.text = "Speciality: $speciality"
    }

    fun setImageProfileImageView(storageRef : StorageReference, context : Context?){
        Glide.with(context!!).load(storageRef)
                .placeholder(R.drawable.profile)
                .into(mImageProfile)
    }
}
